.github/workflows/preview.yml

```
@@ -0,0 +1,85 @@
+      - uses: pnpm/action-setup@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: pnpm
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Override worker name for preview
+        run: sed -i 's/"name": "amibeinganthrottled-com"/"name": "'$WORKER_NAME'"/' wrangler.jsonc
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)